companies/justizvollzugsanstalt-frankfurt-am-main-iii.json

```diff
@@ -0,0 +1,30 @@
+{
+    "slug": "justizvollzugsanstalt-frankfurt-am-main-iii",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "public body"
+    ],
+    "name": "Justizvollzugsanstalt Frankfurt am Main III",
+    "address": "Obere Kreuzäckerstraße 4\n60435 Frankfurt am Main\nDeutschland",
+    "phone": "+49 69 136706",
+    "email": "poststelle@jva-frankfurt3.justiz.hessen.de",
+    "web": "https://justizvollzug.hessen.de/JVA-Frankfurt-III",
+    "sources": [
+        "https://justizvollzug.hessen.de/datenschutz",
+        "https://github.com/datenanfragen/data/pull/1634"
+    ],
+    "required-elements": [
```

Comment:
Do we know where these came from? Are they a guess?

Reply:
I'm sorry, I don't understand. What do you mean?

Reply:
Oh sorry. I meant whether we know how the required elements were determined. Did the person who submitted this guess which elements are necessary? Or maybe they know that from a previous requests/the privacy policy/…

Reply:
Seems like the default guess to me, but I don't know for sure

Reply:
That's one of these cases with community suggestions where it's unfortunately hard to determine what the right thing to do is. :/ I for one really can't adequately guess what reasonable required elements for correctional facilities are and if the submitter also just guessed, it might be better to remove them and just fall back to the defaults?

Reply:
Agreed. As (afaik) we don't have much/any experience with requests to correctional facilites, I'd suggest falling back to the defaults for now. We'll be able to revise the required elements once we have more data on this.